contracts/FeeSettings.sol

```diff
@@ -317,37 +269,13 @@ contract FeeSettings is
      * @param _fees The fees to check
      */
     function checkFeeLimits(Fees memory _fees) internal pure {
+        require(_fees.tokenFeeNumerator <= MAX_TOKEN_FEE_NUMERATOR, "Token fee must be equal or less 5%");
```

Comment:
"or less **than** 5%" -> minor grammar check

Reply:
fixed [0a12b89](https://github.com/corpus-io/tokenize.it-smart-contracts/pull/300/commits/0a12b89a185465a84473fa2c769d80bd41874d3c)